### DISCUSSION TITLE
Authentication with google or slack doesn't work

### DISCUSSION THREAD

**Discussion by devenpandey-airmatrix** [requarks/wiki#4350] [Error / Bug Report]
We followed all the steps to authenticate wiki.js deployed on ec2 instance to login through slack as well as through google but both errors out with authentication failed.

One of the strange thing I noticed is that the admin page it shows slack has logged in however attempt to login fails all the time. 

Both slack and google login attempts have failed and we currently are relying on Local Authentication

Thanks
Devendra

**Comment by omenos** [comment]
Re: Slack

The module used for Slack authentication doesn't support a required feature of Slack OAuth.

https://github.com/Requarks/wiki/discussions/3335

https://github.com/Requarks/wiki/issues/3149
